Source/DafnyCore/AST/DafnyAst.cs

```diff
@@ -769,7 +769,7 @@ public class ActualBindings {
       ArgumentBindings = actuals.ConvertAll(actual => new ActualBinding(null, actual));
     }
 
-    private List<Expression> arguments; // set by ResolveActualParameters during resolution
+    public List<Expression> arguments; // set by ResolveActualParameters during resolution
```

Comment:
`ActualBindings` tries hard to keep this private, so I'm guessing it should stay that way.

Reply:
Done. I now make use of the publicly available `AcceptArgumentExpressionsAsExactParameterList()` instead.